Lib/ast.py

```diff
@@ -812,10 +820,10 @@ def __FunctionDef_helper(self, node, fill_suffix):
         for deco in node.decorator_list:
             self.fill("@")
             self.traverse(deco)
-        def_str = fill_suffix + " " + node.name + "("
+        def_str = fill_suffix + " " + node.name
         self.fill(def_str)
-        self.traverse(node.args)
-        self.write(")")
+        with self.delimit("()"):
```

Comment:
Wait. I'm not sure about this one. Are you going to omit parenthesis here sometimes?

The change is correct, but direct write() calls are maybe better.